src/os/bluestore/BitMapAllocator.h

```diff
@@ -42,6 +42,7 @@ class BitMapAllocator : public Allocator {
     uint64_t want_size, uint64_t alloc_unit, int64_t hint,
     uint64_t *offset, uint32_t *length);
 
+  using Allocator::alloc_extents;
```

Comment:
The root cause for the warning is double virtual declaration for alloc_extents methods in Allocator.h
In fact there is no much sence in declaring following method virtual
 virtual int alloc_extents(uint64_t want_size, uint64_t alloc_unit,
                            int64_t hint, std::vector<AllocExtent> *extents,
                            int *count) {
    return alloc_extents(want_size, alloc_unit, want_size, hint, extents, count);
  }

To be honest I don't see much sense in having it at all - IMHO it rather complicates Allocator interface,
Anyway if for some reason you prefer to have it than removing virtual declaration ( or renaming the method ) should effectively eliminate the warning.